_maps/RandomRuins/SpaceRuins/cryocontainment.dmm

```diff
@@ -0,0 +1,128 @@
+"ab" = (/obj/machinery/atmospherics/pipe/simple/general{dir = 4},/obj/structure/cable{icon_state = "4-8"},/obj/structure/cable{icon_state = "1-8"},/turf/open/floor/plating/airless,/area/ruin/unpowered)
+"al" = (/obj/structure/cable{icon_state = "4-8"},/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4{dir = 4},/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2{dir = 4},/turf/open/floor/plating/airless,/area/ruin/unpowered)
+"bm" = (/obj/structure/table/reinforced,/obj/machinery/button/door{id = cryocontainshutter; name = "Window Shutter Control"; pixel_x = 8; pixel_y = -4},/turf/open/floor/plasteel/airless/dark,/area/ruin/unpowered)
```

Comment:
```suggestion
"bm" = (/obj/structure/table/reinforced,/obj/machinery/button/door{id = "cryocontainshutter"; name = "Window Shutter Control"; pixel_x = 8; pixel_y = -4},/turf/open/floor/plasteel/airless/dark,/area/ruin/unpowered)
```

Reply:
idk why I even bothered to make these functional lmao